package.json

```diff
@@ -135,8 +135,10 @@
     "graph-types": "graphql-codegen --config ./graph-types.codegen.yaml",    
     "lingui:extract": "lingui extract",
     "lingui:compile": "lingui compile",
+    "lingui:branch:main": "git submodule set-branch --branch main src/locales/translations/",
+    "lingui:fetch": "git submodule update --init --remote src/locales/translations",
     "typechain:build": "yarn run typechain --target ethers-v5 --out-dir src/typechain src/abi/*.json src/abi/**/*.json",
-    "preinstall": "[ ! -f src/locales/translations/.git ] && git submodule deinit --all -f; git submodule update --init --recursive --remote",
+    "preinstall": "[ ! -e src/locales/translations/.git ] && yarn lingui:fetch; exit 0",
```

Comment:
i checked out `yarn && yarn build` and `yarn && yarn start` 
on fresh clones and with conflicting file paths
everything works well 
👍

Reply:
The only thing I'm not sure about is this https://github.com/OlympusDAO/olympus-frontend/pull/719#discussion_r753813933